geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/services/StripedExecutorService.java

```diff
@@ -0,0 +1,518 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```

Comment:
Since this file came from a third party, I think you need to leave their license in place at the top of the file - https://www.apache.org/legal/src-headers.html#3party

Reply:
Oops - I think IJ replaced the original license when I added it to the project.

Reply:
Replaced all original license text.